decidim-core/app/cells/decidim/amendable/announcement_cell.rb

```diff
@@ -6,16 +6,13 @@ class AnnouncementCell < Decidim::ViewModel
     include Decidim::ApplicationHelper
 
     def show
-      cell "decidim/announcement", announcement
+      cell "decidim/announcement", announcement, callout_class: state_classes
```

Comment:
These changes seem unrelated to the rest of the PR, are they necessary?

Reply:
After i have implemented the parallel_tests, the pipeline started to fail with"
![image](https://user-images.githubusercontent.com/105683/151516458-91fc0756-74b0-4f1a-bc20-d204f0b56f4a.png)

And while investigating the cause, i have realized that initializer requires ( cell name, model, options = {} ) and we were sending just the Cell and options.

Reply:
Ah ok, great!